doc/tutorial/statistical_inference/unsupervised_learning.rst

```diff
@@ -93,28 +78,21 @@ algorithms. The simplest clustering algorithm is :ref:`k_means`.
     	>>> face_compressed = np.choose(labels, values)
     	>>> face_compressed.shape = face.shape
 
-
-    .. figure:: /auto_examples/cluster/images/sphx_glr_plot_face_compress_001.png
-       :target: ../../auto_examples/cluster/plot_face_compress.html
-
        **Raw image**
```

Comment:
This headline now gets "sucked into" the code example above. My rST foo is not strong enough to suggest a fix :(